### PR TITLE
Some fix

### PR DESCRIPTION
**Pull request by ITJesse**
1. Prevent double click like button.
2. Fix laggy login page animate.
3. Fix MyPage user avatar position.

### CODE REVIEW

File: src/components/UserCover.js [hunk 1]
```diff
@@ -6,11 +6,11 @@ import { globalStyleVariables } from '../styles';
 
 const styles = StyleSheet.create({
   avatarContainer: {
-    height: globalStyleVariables.DRAWER_WIDTH * 9 / 16,
+    height: 172,
```
Comment (alphasp) [inline src/components/UserCover.js]: what is the reason to fix this value here?

File: src/containers/WalkthroughIllustList.js [hunk 2]
```diff
@@ -41,25 +45,34 @@ class WalkthroughIllustList extends Component {
...
+            animated: false,
+            offset: (this.scrollOffset += 30 / this.fps),
+          });
+        }
+        this.autoScrollTimer = requestAnimationFrame(() => {
+          const offset = Date.now() - this.last;
+          this.tmpFps += 1;
+          if (offset >= 1000) {
+            this.fps = this.tmpFps / offset * 1000;
+            console.log(this.fps);
```
Comment (alphasp) [inline src/containers/WalkthroughIllustList.js]: please remove console.log

File: src/components/UserCover.js (hunk 1 above)
Comment (ITJesse) [reply src/components/UserCover.js]: The height here  is too high for the landscape layout on iPad.

File: src/containers/WalkthroughIllustList.js (hunk 2 above)
Comment (ITJesse) [reply src/containers/WalkthroughIllustList.js]: Done.

File: src/components/UserCover.js (hunk 1 above)
Comment (alphasp) [reply src/components/UserCover.js]: The codebase currently only support for portrait mode. 
In app change orientation is not properly supported yet, for example, when you change orientation in app, the width of illust items are not adjust to fully fit for the window width, some code changes need to be done to proper support landscape mode.
For now, you can revert this height value then I will merge this PR.

Comment (ITJesse) [reply src/components/UserCover.js]: Well, in my opinion, the landscape layout is necessary, especially on the iPad.
For now I will remove that, and then find something we can do to deal with the rotate bug...

Comment (ITJesse) [reply src/components/UserCover.js]: Done.

**Comment by alphasp** [comment]
thanks

**Pull request merged**